packages/superset-ui-legacy-preset-chart-nvd3/src/utils.js

```diff
@@ -295,7 +295,7 @@ export function wrapTooltip(chart, maxWidth) {
 
 export function tipFactory(layer) {
   return d3tip()
-    .attr('class', 'd3-tip')
+    .attr('class', `d3-tip ${layer.annotationTipClass}`)
```

Comment:
Better use `{layer.annotationTipClass || ''}` so we don't add an "undefined" class to the tip.

Reply:
fixed.